msgraphforentra_connector.py

```diff
@@ -0,0 +1,1631 @@
+# File: msgraphforentra_connector.py
+#
+# Copyright (c) 2022-2024 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+#
+#
+
+import phantom.app as phantom
+from phantom.action_result import ActionResult
+from phantom.base_connector import BaseConnector
+
+try:
+    from urllib.parse import quote, urlencode
+except Exception:
+    from urllib import quote, urlencode
+
+import ast
+import grp
+import json
+import os
+import pwd
+import re
+import time
+from datetime import datetime, timedelta
+
+import encryption_helper
+import msal
+import requests
+from bs4 import BeautifulSoup
+from django.http import HttpResponse
+
+from msgraphforentra_consts import *
```

Comment:
I actually _hate_ this recommendation from our own app wizard and standards, and I will be eradicating it ASAP. Wildcard imports are a terrible idea
```suggestion
import msgraphforentra_consts as consts
```

Reply:
no argument here!